sdk/python/feast/type_map.py

```diff
@@ -372,11 +372,15 @@ def _python_value_to_proto_value(
                 if feast_value_type == ValueType.BYTES_LIST:
                     raise _type_err(sample, ValueType.BYTES_LIST)
 
-                json_value = json.loads(sample)
-                if isinstance(json_value, list):
+                json_sample = json.loads(sample)
```

Comment:
Any chance you can add a test here?

Reply:
Hi, sure thing. Did you have in mind something to check that `json.loads(sample)` does not error out? E.g. if there's something like a json decode error?

Reply:
Actually, it's fine. There's no test there currently so no need to add additional work here. Thank you!